src/styles/sidebar/components/selection-tabs.scss

```diff
@@ -28,6 +30,10 @@
   min-height: 18px;
 
   user-select: none;
+
+  &:hover {
+    color: var.$link-color-hover;
```

Comment:
This preserves the existing behavior, although we might want to consider dropping it entirely as non-link actions elsewhere in the app generally don't have this hover effect.

Reply:
agreed